### DISCUSSION TITLE
[灌水] 天马速运，丢货专家！

### DISCUSSION THREAD

**Discussion by xaws46** [share121/inter-knot#1339] [其他]
![32bcd90d1a60b05156b9c62777e4349c65256627](https://github.com/user-attachments/assets/c3a90c39-b23a-4115-be61-a78788146c2c)
早上货车还停在这里，空洞一来一去再看就只剩个贴纸了。估计很快就有新委托发布了，想接的别又让脚本抢先了。

**Comment by lllliufeng** [comment]
来了。
不过反向宣传哈哈哈，不敢用天马速运送快递了

**Comment by a-132-NIDIE** [comment]
师傅的拉面上次就是这么丢的（怒）

**Comment by gaojiaobei** [comment]
不愧是天马速运啊
